docs/docs/guides/build/external-pipelines/scalaspark-pipeline.md

```diff
@@ -0,0 +1,145 @@
+---
+title: "Build pipelines with Scala and Spark"
+description: "Learn to integrate Dagster Pipes with Scala and Spark to orchestrate Spark and Scala jobs in a Dagster pipeline."
+sidebar_position: 71
+---
+
+This tutorial is focused on using Dagster Pipes to launch & monitor Apache Spark jobs implemented in Scala. The [Spark integration page](/integrations/libraries/spark) provides more information on using Pipes with specific Spark providers, such as AWS EMR or Databricks.
+
+Spark is often used with object stores such as Amazon S3. In our example, Dagster will use an S3
+bucket to communicate with Apache Spark (sending data to the Spark application, as well as reading logs and results
+from Spark).
+
+## Prerequisites
+
+- An [AWS S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/GetStartedWithS3.html) to be used for communication between Spark and Dagster (and the corresponding `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`).
+- We are going to use [Docker](https://docs.docker.com/get-started/get-docker/) to emulate a typical Spark setup locally.
+  This includes Apache Spark and the Java SDK, as well as the required Dagster libraries to run a development orchestration server.
```

Comment:
"This" is a bit unclear; "This setup" perhaps?

(Otherwise, it sounds like "This" refers to Docker IMO.)